debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorFilterIT.java

```diff
@@ -69,41 +68,41 @@ public void before() throws SQLException {
                 "GRANT CREATE TABLE TO debezium2",
                 "GRANT CREATE SEQUENCE TO debezium2",
                 "ALTER USER debezium2 QUOTA 100M ON users",
-                "create table debezium2.table2 (id numeric(9,0) not null,name varchar2(1000),primary key (id))",
-                "create table debezium2.nopk (id numeric(9,0) not null)",
-                "GRANT ALL PRIVILEGES ON debezium2.table2 to debezium",
-                "GRANT SELECT ON debezium2.table2 to " + TestHelper.CONNECTOR_USER,
-                "GRANT SELECT ON debezium2.nopk to " + TestHelper.CONNECTOR_USER,
+                "CREATE TABLE debezium2.table2 (id NUMERIC(9,0) NOT NULL, name VARCHAR2(1000), PRIMARY KEY (id))",
```

Comment:
Very thorough :)

Reply:
When the tests didn't work for me I was trying to understand what's going on and thought it would be my code changes. To clearer understand what's going on I did that.
Finally it turned out I just didn't re-setup the docker container often enough inside the vagrant VM LOL.